tests/start_proxy/start_proxy_test.py

```diff
@@ -17,10 +17,10 @@
 
 import os, inspect
 
-currentdir = os.path.dirname(
-    os.path.abspath(inspect.getfile(inspect.currentframe())))
-sys.path.insert(0, currentdir + "/../../docker/generic")
-from start_proxy import gen_bootstrap_conf, make_argparser, gen_proxy_config, gen_envoy_args
+# currentdir = os.path.dirname(
+#     os.path.abspath(inspect.getfile(inspect.currentframe())))
+# sys.path.insert(0, currentdir + "/../../docker/generic")
```

Comment:
if importing "docker.generic.start_proxy" works, then remove above 3 lines then.

Reply:
Actually, I mistakenly left it like this due to problems with my environment, I will just change it back - I don't want to perform any unnecessary changes.